keyboards/mino_plus/soldered/info.json

```diff
@@ -0,0 +1,26 @@
+{
+    "manufacturer": "ShandonCodes",
+    "keyboard_name": "Mino Plus",
+    "maintainer": "ShandonCodes",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["B4", "B3", "A15", "A10", "A8", "B14", "B12", "B10", "A5", "A4", "A3", "B0", "A7", "C15", "B5"],
+        "rows": ["B9", "B8", "A2", "A1", "A0"]
+    },
+    "processor": "STM32F072",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0003",
+        "vid": "0x7877"
+    }
```

Comment:
You're missing the necessary layout data for use with the QMK configurator here.
(eg. [here](https://docs.qmk.fm/#/porting_your_keyboard_to_qmk?id=layout-macros))

Reply:
@waffle87 Added layout data.